src/app/tests/suites/certification/Test_TC_CC_6_2.yaml

```diff
@@ -59,13 +61,54 @@ tests:
               - name: "optionsOverride"
                 value: 0
 
-    - label: "Read current color temprature"
+    - label: "Read current color temprature attribute from DUT several times"
       disabled: true
       command: "readAttribute"
       attribute: "color temperature"
       response:
           value: -1
 
+    - label: "Move down color temperature command"
+      command: "MoveColorTemperature"
+      arguments:
+          values:
+              - name: "moveMode"
+                value: 3
+              - name: "rate"
+                value: 20
+              - name: "colorTemperatureMinimum"
+                value: 1
+              - name: "colorTemperatureMaximum"
+                value: 255
+              - name: "optionsMask"
+                value: 0
+              - name: "optionsOverride"
+                value: 0
+
+    - label: "Read current color temprature attribute from DUT several times"
+      disabled: true
```

Comment:
Why is this disabled?  Shouldn't we just wait, then read it?

Reply:
Reading DUT several times still not implemented in YAML so we disabled it

Reply:
I'm not sure what "reading several times not implemented"  You can do sereral read steps....

Reply:
I think what he means is that I believe we don't have a way to check if the value is greater than or less than a saved value in previous step yet. If its there we can just implement timeout for 30 secs , read if greater than the previous read value for x times as needed by the TC

Reply:
Ah, we can test >= with minValue constraints, I'd think, but not strictly greater.....  OK, thanks!

Reply:
https://github.com/project-chip/connectedhomeip/issues/11234 was raised previously for this ,we need to read the value of the attribute overtime and make sure it is incrementing gradually to he desired value.
reading the value in multiple steps might be challenging  as we might not get constant value on each run